src/openlcb/MultiConfiguredPC.hxx

```diff
@@ -138,10 +138,20 @@ public:
         ConfigUpdateService::instance()->register_update_listener(this);
         producedEvents_ = new EventId[size * 2];
         std::allocator<debouncer_type> alloc;
+// C++20 standard removed the construct and destruct methods from
+// std::allocator which are now available in std::allocator_traits requiring
+// passing in the allocator to these methods.
+#if __cplusplus >= 202002L
+        std::allocator_traits<std::allocator<debouncer_type>> alloc_traits;
```

Comment:
alloc_traits is a static class, so we don't need to make an instance. Use this pattern:
```
using atraits = std::allocator_traits<std::allocator<debouncer_type>>;
...
atraits::construct(alloc, ...)
```